org.adempiere.base/src/org/compiere/model/X_M_Attribute.java

```diff
@@ -207,6 +207,8 @@ public int getAD_Val_Rule_ID()
 
 	/** AttributeValueType AD_Reference_ID=326 */
 	public static final int ATTRIBUTEVALUETYPE_AD_Reference_ID=326;
+	/** Chosen Multiple Selection List = CL */
+	public static final String ATTRIBUTEVALUETYPE_ChosenMultipleSelectionList = "C";
```

Comment:
It seems you are missing the saveSelection changes for ATTRIBUTEVALUETYPE_ChosenMultipleSelectionList